README.md

```diff
@@ -769,6 +769,11 @@ based session store. Supports all backends supported by Fortune (MongoDB, Redis,
 [nedb-session-store-url]: https://www.npmjs.com/package/nedb-session-store
 [nedb-session-store-image]: https://badgen.net/github/stars/JamesMGreene/nedb-session-store?label=%E2%98%85
 
+[![★][prisma-session-store-image] prisma-session-store][prisma-session-store-url] A session store for the [Prisma Framework](https://www.prisma.io).
```

Comment:
Linting error is that these link names should match the name of npm module in order to ensure they are unique. Since the npm module here is `@quixo3/prisma-session-store` it is expecting the link names to start with that too.

Reply:
>>Linting error is that these link names should match the name of npm module in order to ensure they are unique.

Got it. Ok; I think I've fixed this [here](https://github.com/expressjs/session/pull/760/commits/2c138da6c6e374fd37c520908b54587b73c02ff7l); see what you think.